app/models/planet.py

```diff
@@ -0,0 +1,33 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
+    life = db.Column(db.Boolean)
+    moons = db.Column(db.Integer)
+
+    def to_dict(self):
+        return dict(
+            id=self.id,
+            name=self.name,
+            description=self.description,
+            life=self.life,
+            moons=self.moons,
+        )
+    
+    @classmethod
+    def from_dict(cls, data_dict):
+        return cls(
+            name=data_dict["name"],
+            description=data_dict["description"],
+            life=data_dict["life"],
+            moons=data_dict["moons"]
+        )
+    
+    def replace_details(self, data_dict):
+        self.name=data_dict["name"]
+        self.description=data_dict["description"]
+        self.life=data_dict["life"]
+        self.moons=data_dict["moons"]
+        return self.to_dict()
```

Comment:
Having this method return the dictionary form of the record feels a little premature to me. Yes, it's very likely that we'll want this information, but mutating method like this usually just perform the modifications as their single responsibility. They might return the instance itself (it can be useful for chaining operations), but I'd leave it to the caller to decide whether they want to convert the result to a dictionary themself.